inst/stan/estimate_infections.stan

```diff
@@ -187,28 +188,32 @@ generated quantities {
   vector[return_likelihood ? ot : 0] log_lik;
   if (estimate_r){
     // estimate growth from estimated Rt
-    real set_gt_mean = (gt_mean_sd[1] > 0 ? gt_mean[1] : gt_mean_mean[1]);
-    real set_gt_sd = (gt_sd_sd [1]> 0 ? gt_sd[1] : gt_sd_mean[1]);
-    r = R_to_growth(R, set_gt_mean, set_gt_sd);
+    real gen_gt_mean = pmf_mean(gt_rev_pmf, 1, 1);
+    real gen_gt_var = pmf_var(gt_rev_pmf, 1, 1, gen_gt_mean);
+    r = R_to_growth(R, gen_gt_mean, gen_gt_var);
   } else {
     // sample generation time
-    real gt_mean_sample[1];
-    real gt_sd_sample[1];
-    vector[gt_max[1]] gt_rev_pmf;
+    real gt_mean_sample[gt_n_p];
+    real gt_sd_sample[gt_n_p];
+    vector[gt_max_total] gen_rev_pmf;
```

Comment:
I am bit lost why we are being inconsistent with prefixs here? Happy to just call everything `gen_` vs switching?